### ISSUE TITLE
ordinary user can not query system meta table like user_tab_columns

### ISSUE THREAD

**Issue by marktanzy** [sijms/go-ora#326]
`system` user can query as `select * from user_tab_columns` but user `goora` can not, it aways blocked in the `conn.QueryContext(context.Background(), querySql)`

my query code as following:

```go
package main

import (
	"context"
	"database/sql"
	"fmt"
	"testing"

	_ "github.com/sijms/go-ora/v2"

func TestMa(t *testing.T) {
	user := "goora"
	password := "goora"
	host := "localhost"
	port := "1521"
	sid := "EE"

	openString := fmt.Sprintf("%s/%s@%s:%s?sid=%s",
		user, password, host, port, sid)
	conn, err := sql.Open("oci8", openString)
	if err != nil {
		fmt.Printf("Open error is not nil: %v", err)
		return
	}
	if conn == nil {
		fmt.Println("db is nil")
		return
	}

	querySql := "select * from USER_TAB_COLUMNS"
	rows, err := conn.QueryContext(context.Background(), querySql)  // bloking .......
	if err != nil && err != sql.ErrNoRows {
		fmt.Println("00000000000", err.Error())
		return
	}
	fmt.Println("111111111")
	if err == sql.ErrNoRows {
		return
	}

	cols, err := rows.ColumnTypes()
	if err != nil {
		return
	}
	for _, col := range cols {
		println(col)
	}
	defer func() { _ = rows.Close() }()
	return
}
```

what is my problem?

**Comment by sijms** [comment]
would you replace `*` by columns name and try again

**Comment by sijms** [comment]
there is a pug found when there is a LONG column I fix it in v2.6.7
would you test again
